paderbox/math/vector.py

```diff
@@ -25,7 +25,10 @@ def cos_distance(a, b):
     :param b: vector b (1xN or Nx1 numpy array)
     :return: distance (scalar)
     """
-    return 0.5 * (1 - sum(a * b) / np.sqrt(sum(a ** 2) * sum(b ** 2)))
+    assert a.shape == b.shape, 'Both vectors must have the same dimension'
```

Comment:
`assert a.shape == b.shape, f'Both vectors must have the same dimension. {a.shape} {b.shape}'`